.circleci/config.yml

```diff
@@ -61,6 +61,11 @@ workflows:
           requires:
             - build
           context: sonarcloud
+      - BitBucket-tests:
```

Comment:
Although it's not a big deal, I'm gonna advocate for keeping the capitalization consistent (lowercase) here.

Reply:
I have changed it from `BitBucket-tests` to `bitbucket-tests`